src/main/java/org/kpmp/repositoryDataset/RepositoryDatasetService.java

```diff
@@ -55,10 +55,44 @@ public RepositoryDatasetService(
 		this.env = env;
 	}
 
-  public List<RepositoryDataset> getRepositoryDataset() throws Exception {
-    List <RepositoryDataset> datasets = new ArrayList<>();
+  public List<RepositoryDatasetDisplay> getRepositoryDataset() throws Exception {
+    List <RepositoryFileDataset> datasets = new ArrayList<>(); 
+    String maxReleaseVersion = fileRepo.max();
     datasets.addAll(fileRepo.findAll());
-    return datasets;
+    Map<String, RepositoryDatasetDisplay> displayFiles = new HashMap<>();
+	for (RepositoryFileDataset repositoryDataset : datasets) {
+        if (displayFiles.containsKey(repositoryDataset.getId().getDlFileId())) {
+            if(repositoryDataset.getId().getDlFileId().equalsIgnoreCase("E283EACF-7950-4165-85E2-CE0128CDAA26")){
+                System.out.println("Adding redcapId " + repositoryDataset.getId().getRedcapId());
```

Comment:
Let's get rid of the sysout, and this if statement...this looks like it was added for testing